keda/README.md

```diff
@@ -111,11 +111,9 @@ their default values.
 | `priorityClassName` | string | `""` | priorityClassName for all KEDA components |
 | `rbac.aggregateToDefaultRoles` | bool | `false` | Specifies whether RBAC for CRDs should be [aggregated](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles) to default roles (view, edit, admin) |
 | `rbac.create` | bool | `true` | Specifies whether RBAC should be used |
+| `rbac.enabledCustomScaledRefKinds` | bool | `true` | Whether rbac for unseen crds that can have a scale subresource should be created |
+| `rbac.scaledRefKinds` | string | `nil` | List of custom resources that support the scale subresource that can be referenced by `scaledobject.spec.scaleTargetRef` the feature needs to be also enabled by `enabledCustomScaledRefKinds` If left empty, rbac for `apiGroups: *` & `resources: */scale` will be created note: Deployments and StatefulSets are supported out of the box |
```

Comment:
nit: minor formatting tweaks
```suggestion
| `rbac.scaledRefKinds` | string | `nil` | List of custom resources that support the `scale` subresource and can be referenced by `scaledobject.spec.scaleTargetRef`. The feature needs to be also enabled by `enabledCustomScaledRefKinds`. If left empty, RBAC for `apiGroups: *` and `resources: */scale` will be created.
note: `Deployments` and `StatefulSets` are always enabled |
```

Reply:
This file is selfgenerated via helm-docs  from values.yaml, could you update it directly?

Reply:
he made (/suggested and i accepted via button) the changes to both actually. I've also double checked and re-run helm-docs locally and it didn't detect any change

Reply:
he made (well, suggested and i accepted) the change actualy to both files and I checked that locally by running the helm-docs (there was no drift)